lib/katello/tasks/migrate_content_stats.rake

```diff
@@ -0,0 +1,21 @@
+namespace :katello do
+  desc "Retrieve Pulp 2 -> Pulp 3 content migration stats"
+  task :migrate_content_stats => [:environment] do
```

Comment:
Based on the name of the pulp3 migration task: `pulp3_migration` would it be more clear and consistent to name this task `pulp3_migration_stats` ?

Reply:
sure, that sounds good to me.